solidity/README.md

```diff
@@ -1,26 +1,26 @@
-﻿Solidity contract workflow
+﻿## Solidity contract workflow
 
 Start with source code of the contract in <MyContract.sol>
 
 1) Compile Solidity source to TVM assembler: 
 
-solc --tvm MyContract.sol >MyContract.code
+*solc --tvm MyContract.sol >MyContract.code*
```

Comment:
it is better to wrap the code with [special tags for the code](https://help.github.com/en/articles/creating-and-highlighting-code-blocks)

Reply:
Thanks for the tip! Looks much better. Updated.